src/ngraph/runtime/gpu/gpu_memory_manager.cpp

```diff
@@ -86,17 +115,16 @@ size_t runtime::gpu::GPUAllocator::reserve_argspace(const void* data, size_t siz
     // add parameter data to host buffer that will be transfered to device
     size = pass::MemoryManager::align(size, runtime::gpu::GPUMemoryManager::alignment);
     size_t offset = m_manager->queue_for_transfer(data, size);
-    // required to capture m_manager pointer
-    // directly rather than `this` pointer
-    auto manager = m_manager;
+    auto local = --m_manager->m_argspace_mem.end();
```

Comment:
nit: use `.back()`?

Reply:
`local` needs to be an iterator to the specific element in the list which will be lazily updated prior to the memory primitive being invoked. However, I updated it to use std::prev instead of operator--() for readability.